src/geometry/ArborX_DetailsAlgorithms.hpp

```diff
@@ -302,6 +304,16 @@ struct expand<BoxTag, PointTag, Box, Point>
   }
 };
 
+// expand a box to include a point
+template <typename Box, typename KDOP>
+struct expand<BoxTag, KDOPTag, Box, KDOP>
+{
+  KOKKOS_FUNCTION static void apply(Box &box, KDOP const &kdop)
+  {
+    Details::expand(box, (ArborX::Box)kdop);
```

Comment:
This looks more like a tmp hack.  Please annotate with a FIXME

Reply:
It's not really a hack. I have no immediate plans to dimensionalize kdop. But I do want to have `expand(box, kdop)`. So, I'm not sure what FIXME you would want here, as the current version is correct for the current version of kdop.